A2 - Loops & Literals/first.rb

```diff
@@ -0,0 +1,32 @@
+# Returns sum of values in list
+def sum(list)
+  list.inject(0){|sum,x| sum += x }
```

Comment:
Nice trick! Though, is there a way to default the sum to 0 if the list is empty? Because I have empty lists as well that need to return a sum.